tests/TestCase/Validation/ValidationTest.php

```diff
@@ -1483,6 +1483,41 @@ public function testDateY()
     }
 
     /**
+     * test date validation when passing an array
+     *
+     * @return void
+     */
+    public function testDateArray() {
+        $date = ['year' => 2014, 'month' => 2, 'day' => 14];
+        $this->assertTrue(Validation::date($date));
+        $date = ['year' => 'farts', 'month' => 'derp', 'day' => 'farts'];
+        $this->assertFalse(Validation::date($date));
+    }
```

Comment:
Won't work. But I think that is fair since you are not sending a string but an array, so the default would be fine.